### PR TITLE
fix: logout dialog ui

### PR DESCRIPTION
**Pull request by lelenei**
## Description

Closes # mf-3672 mf-3692

## Type of change

- [ ] Documentation
- [ ] Code refactoring (Restructuring existing code w/o changing its observable behavior)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (a fix or feature that would make something no longer possible to do/require old user must upgrade their Mask Network to this new version)

## Checklist

- [ ] My code follows the style guidelines of this project.
- [ ] I have performed a self-review of my own code.
  - [ ] I have removed all in development `console.log`s
  - [ ] I have removed all commented code.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have read [Internationalization Guide](https://github.com/DimensionDev/Maskbook/blob/develop/docs/i18n-guide.md) and moved text fields to the i18n JSON file.

If this PR depends on external APIs:

- [ ] I have configured those APIs with CORS headers to let extension requests get passed. 
  - chrome extension: `chrome-extension://[id]`
  - firefox extension: `moz-extension://[id]`
- [ ] I have delegated all web requests to the background service via the internal RPC bridge.

### CODE REVIEW

File: packages/shared-base/src/utils/personas.ts [hunk 1]
```diff
@@ -5,6 +5,11 @@ export function formatPersonaFingerprint(fingerprint: string, size = 0) {
     if (size === 0) return fingerprint
     return `${fingerprint.slice(0, Math.max(0, 2 + size))}...${fingerprint.slice(-size)}`
 }
+export function formatPersonaFingerprint2(fingerprint: string) {
```
Comment (guanbinrui) [inline packages/shared-base/src/utils/personas.ts]: Use `formatPersonaFingerprint` stead.

File: packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx [hunk 2]
```diff
@@ -273,4 +234,111 @@ export const LogoutUI = memo<LogoutUIProps>(
...
+        display: 'grid',
+        gridTemplateColumns: `repeat(${props.length > 1 ? 2 : 1}, 1fr)`,
+        marginTop: 12,
+        padding: 0,
+    },
+    wallet: {
+        padding: '8px',
+        display: 'flex',
+        gap: 8,
+        backgroundColor: '#F7F9FA',
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx]: Avoid dead code color.

File: packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx [hunk 3]
```diff
@@ -273,4 +234,111 @@ export const LogoutUI = memo<LogoutUIProps>(
     },
 )
 
+const useWalletsStyles = makeStyles<{ length: number }>()((theme, props) => ({
+    persona: {
+        padding: '8px 16px',
+        display: 'flex',
+        gap: 20,
+        backgroundColor: '#F7F9FA',
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx]: Avoid dead code color.

File: packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx [hunk 4]
```diff
@@ -273,4 +234,111 @@ export const LogoutUI = memo<LogoutUIProps>(
...
+        borderRadius: 8,
+    },
+    nickname: {
+        lineHeight: '16px',
+        fontWeight: 600,
+        fontSize: 14,
+        color: '#1C68F3',
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx]: Avoid dead code color.

File: packages/dashboard/src/components/PasswordField/index.tsx [hunk 5]
```diff
@@ -16,14 +16,16 @@ const PasswordField = forwardRef((props: PasswordFieldProps, ref: ForwardedRef<a
...
+                            <IconButton
+                                aria-label="toggle password visibility"
+                                onClick={() => setShowPassword(!showPassword)}
+                                onMouseDown={(event) => event.preventDefault()}
+                                edge="end"
+                                size="small">
+                                {showPassword ? <Icons.EyeOff /> : <Icons.Eye />}
+                            </IconButton>
+                        ) : null}
                     </InputAdornment>
```
Comment (nuanyang233) [inline packages/dashboard/src/components/PasswordField/index.tsx]:
```tsx
 endAdornment: show ? (
                    <InputAdornment position="end">
                        <IconButton
                            aria-label="toggle password visibility"
                            onClick={() => setShowPassword(!showPassword)}
                            onMouseDown={(event) => event.preventDefault()}
                            edge="end"
                            size="small">
                            {showPassword ? <Icons.EyeOff /> : <Icons.Eye />}
                        </IconButton>
                    </InputAdornment>
                ) : null,
```

File: packages/dashboard/src/pages/Personas/components/LogoutPersonaDialog/index.tsx [hunk 6]
```diff
@@ -50,40 +82,191 @@ export const LogoutPersonaDialog = memo<LogoutPersonaDialogProps>(
...
+                        variant="contained">
                         {t.personas_logout()}
                     </LoadingButton>
                 </DialogActions>
             </MaskDialog>
         )
     },
 )
+
+const useStyles = makeStyles<{ length: number }>()((theme, props) => ({
```
Comment (guanbinrui) [inline packages/dashboard/src/pages/Personas/components/LogoutPersonaDialog/index.tsx]: Move those newly added components to another file.

File: packages/mask/src/extension/popups/components/PasswordField/index.tsx [hunk 7]
```diff
@@ -15,14 +15,16 @@ export const PasswordField = memo(
...
                         <InputAdornment position="end">
-                            <IconButton
-                                aria-label="toggle password visibility"
-                                onClick={() => setShowPassword(!showPassword)}
-                                onMouseDown={(event) => event.preventDefault()}
-                                edge="end"
-                                size="small">
-                                {showPassword ? <Icons.EyeOff /> : <Icons.Eye />}
-                            </IconButton>
+                            {show ? (
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/components/PasswordField/index.tsx:18]:
```ts
endAdornment: show ? ... : undefined,
```

File: packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx [hunk 8]
```diff
@@ -183,50 +158,35 @@ export const LogoutUI = memo<LogoutUIProps>(
...
-                                i18nKey="popups_persona_disconnect_manage_wallet_warning"
-                                values={{
-                                    addresses: manageWallets.map((x) => formatEthereumAddress(x.address, 4)).join(','),
-                                    persona: selectedPersona?.nickname ?? '',
-                                }}
-                                components={{ span: <Typography component="span" sx={{ wordBreak: 'break-word' }} /> }}
-                            />
-                        </Typography>
+                        <>
+                            <br />
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx]: Use CSS stead.

File: packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx [hunk 9]
```diff
@@ -273,4 +234,111 @@ export const LogoutUI = memo<LogoutUIProps>(
...
+        },
+        lineHeight: 0,
+        marginLeft: 6,
+    },
+}))
+interface ManageWalletProps {
+    manageWallets: Wallet[]
+}
+
+function ManageWallet({ manageWallets }: ManageWalletProps) {
```
Comment (guanbinrui) [inline packages/mask/src/extension/popups/pages/Personas/Logout/index.tsx]: ?

**Review by guanbinrui — APPROVED** [review]

**Pull request merged**